glotaran/builtin/megacomplexes/decay/test/test_spectral_irf.py

```diff
@@ -41,7 +42,7 @@
 {MODEL_BASE}
 irf:
     irf1:
-        type: spectral-gaussian
+        type: gaussian
```

Comment:
Because of api change. If spectral-gaussian there are now more required arguments, which are not needed for just gaussian.